src/main/kotlin/racingcar/domain/Car.kt

```diff
@@ -2,17 +2,29 @@ package racingcar.domain
 
 import racingcar.domain.MovementCommand.FORWARD
 
-class Car {
+class Car(
+    val name: String
+) {
     var position: Int = 0
         private set
 
+    init {
+        require(name.isNotBlank()) { "이름은 공백일 수 없습니다." }
+        require(NAME_LENGTH_RANGE.contains(name.length)) { "이름은 최소 1글자, 최대 5글자이어야 합니다." }
```

Comment:
```suggestion
        require(name.length in NAME_LENGTH_RANGE)
```
이렇게도 쓸 수 있어요. 이 때 `in` 키워드는 작성해주신 contains와 같습니다. `in` 위에 커서를 놓고 cmd + B를 누르면 확인해보실 수 있을 거에요!